woocommerce/google-listings-and-ads
# UX improvement for Google Ads account card on Ads setup page

### Changes proposed in this Pull Request:

Part of UX improvement for Google Ads setup. It's also related to #1065 and based on it.

- Add Google Ads appearance and `appearance.defaultDescription` to `<AccountCard>`.
- Rename **google-ads-account-section** folder to **google-ads-account-card**.
- Adjust the Google Ads account cards with all states to the UI style of UX Improvements project.
- Fix the flash loading state for the ConnectAds.
    ![Kapture 2021-11-19 at 14 45 33](https://user-images.githubusercontent.com/17420811/142577614-284c67c8-6f8b-4e97-b318-0b4257a3188f.gif)

### Screenshots:

#### 🎥 . Authorization, connecting an existing account, and connected account

![Kapture 2021-11-19 at 14 30 59](https://user-images.githubusercontent.com/17420811/142577673-c0d044b5-c08f-407e-b252-caa2c3403933.gif)

#### 📷 . Authorization card

![image](https://user-images.githubusercontent.com/17420811/142579252-21e7b90b-43fd-49dd-87a7-d0789e526ed8.png)

#### 📷 . Creation card with existing accounts

![2021-11-19 14 34 25](https://user-images.githubusercontent.com/17420811/142577695-91e765db-ca53-4908-9aa2-fc9694169aae.png)

#### 📷 . Creation card without existing accounts

![2021-11-19 14 34 46](https://user-images.githubusercontent.com/17420811/142577700-731a0512-b486-4ae0-b7f0-f1fb4633f378.png)

#### 📷 .  Connecting an existing account card

![image](https://user-images.githubusercontent.com/17420811/142579614-ad420669-57bd-4013-837e-9c288af2f475.png)

#### 📷 .  Connected card

![image](https://user-images.githubusercontent.com/17420811/142579543-da350519-98f8-4b24-846a-d868667ce3b8.png)

### Detailed test instructions:

1. Prepare account state if you've finished all accounts setup before.
    1. Go to the disconnection function in the GLA Settings, and finish the **Disconnect Google Ads account only**".
    1. Open the **Connection Test** page, and click on "Disconnect Google Account" button.
    1. Refresh the GLA Settings and reconnect Google account with Site verification and Content API permission scopes (no Google Ads).
1. Go to the Ads setup page, it should show the authorization card
1. Finish the authorization, it should show the connection card if there're any existing Google Ads accounts, otherwise, display the creation card.
1. After connecting/creating Google Ads account, it should show the connected card.

### Changelog entry

> Tweak - Improve UX and adjust UI style for the accounts connection step of the Google Ads & Paid Campaign setup page.
